opentech/apply/models.py

```diff
@@ -100,7 +100,17 @@ class Option(Orderable):
 
 
 class Category(ClusterableModel):
+    """Used to manage the global select questions used in most of the application form
+    Also used in the front end by editors when writing about projects.
```

Comment:
This indicates Category lives in a grey area as it is not 100% in the apply app if used in the front-end by editors :)

Reply:
As discussed. Agreed it does cross a boundary, however in my mind this is the home of the category (it is where is it configured). Other uses will just be views on this data.

Reply:
👍